packages/axes/src/const.ts

```diff
@@ -75,6 +75,8 @@ export const TRANSFORM = (() => {
 })();
 
 export const PREVENT_DRAG_CSSPROPS = {
+  "-webkit-user-select": "none", /* Safari */
+  "-ms-user-select": "none", /* IE 10 and IE 11 */
```

Comment:
It is judged that there will be few users, but how about adding the `-moz-` prefix as well?

Reply:
Considering the old version of Firefox, adding `user-select` with `-moz-` prefix should be better.